docs/environment-variables.md

```diff
@@ -68,3 +68,16 @@ When `1`, debug information about coverage behavior is printed to stderr.
 
 When `1`, debug information from gazelle is printed to stderr.
 :::
+
+:::{envvar} RULES_PYTHON_VENVS_ROOT
```

Comment:
Just wondering if `RULES_PYTHON_VENVS_ROOT` is slightly ambiguous. This is the bootstrap venv and not a venv that a user can benefit from. Or can they? If I run something like:
```
RULES_PYTHON_VENVS_ROOT=${PWD}/.venv bazel run //:foo_binary -- --help
```

Then I would have an interpreter/venv that I can use to run `foo_binary` later or point it to the editor? My current understanding is that it is not the case.

Maybe having `RULES_PYTHON_BOOTSTRAP_VENV_ROOT` could be useful?

Another slightly related question - why are we using plural?